Files/Dialogs/FilesystemOperationDialog.xaml.cs

```diff
@@ -23,6 +26,65 @@ public FilesystemOperationDialog(FilesystemOperationDialogViewModel viewModel)
             ViewModel = viewModel;
             ViewModel.View = this;
         }
+
+        private void MenuFlyoutItem_Click(object sender, Windows.UI.Xaml.RoutedEventArgs e)
+        {
+            var t = (sender as MenuFlyoutItem).Tag as string;
+            if(t == "All")
+            {
+                var i = DetailsGrid.SelectedItems.FirstOrDefault() as FilesystemOperationItemViewModel;
+                if(i is not null)
+                {
+                    ViewModel.ApplyConflictOptionToAll(i.ConflictResolveOption);
+                }
+                return;
+            }
+
+            var op = (FileNameConflictResolveOptionType)int.Parse(t);
+            foreach (var item in DetailsGrid.SelectedItems)
+            {
+                if(item is FilesystemOperationItemViewModel model)
+                {
+                    model.TakeAction(op);
+                }
+            }
+        }
+
+        private void MenuFlyout_Opening(object sender, object e)
+        {
+            if(!ViewModel.MustResolveConflicts)
+            {
+                return;
+            }
+
+            if (((sender as MenuFlyout)?.Target as ListViewItem)?.Content is FilesystemOperationItemViewModel li)
+            {
+                if(!DetailsGrid.SelectedItems.Contains(li))
+                {
+                    DetailsGrid.SelectedItems.Add(li);
+                }
+            }
+
+            if (DetailsGrid.SelectedItems.Count == 1 && DetailsGrid.SelectedItems.Any(x => (x as FilesystemOperationItemViewModel).ActionTaken))
```

Comment:
@winston-de what do you think about adding this to not giving the "Apply to all" option when only copying one item?:

```suggestion
            if (DetailsGrid.Items.Count > 1 && DetailsGrid.SelectedItems.Count == 1 && DetailsGrid.SelectedItems.Any(x => (x as FilesystemOperationItemViewModel).ActionTaken))
```